styles/app.scss

```diff
@@ -4,3 +4,226 @@
 // $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 // @import 'font-awesome';
 @import 'box-sizing';
+@import '__variables';
+@import '__side-bar';
+
+body {
+  margin: 0;
+  width: 100vw;
```

Comment:
This is making the screen have a side scrolly bit that is a bit weird. Otherwise 👍